library/src/main/java/com/google/android/exoplayer/text/subrip/SubripCue.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.exoplayer.text.subrip;
+
+import android.text.Layout.Alignment;
+
+import com.google.android.exoplayer.C;
+import com.google.android.exoplayer.text.Cue;
+
+/**
+ * A representation of a SubRip cue.
+ */
+/* package */ final class SubripCue extends Cue {
+
+  public final long startTime;
+  public final long endTime;
+
+  public final long x1;
+  public final long x2;
+  public final long y1;
+  public final long y2;
+
+  public SubripCue(CharSequence text) {
+    this(Cue.UNSET_VALUE, Cue.UNSET_VALUE, Cue.UNSET_VALUE,text);
+  }
+
+  public SubripCue(long startTime, long endTime, int position, CharSequence text) {
+    this(startTime, endTime, text, Cue.UNSET_VALUE, Cue.UNSET_VALUE, Cue.UNSET_VALUE,
+            Cue.UNSET_VALUE, Cue.UNSET_VALUE, position, null, Cue.UNSET_VALUE);
+  }
+
+  public SubripCue(long startTime, long endTime, CharSequence text, long x1, long x2, long y1,
```

Comment:
Hm, those fields and parameters should come back, because SubRip supports them.
This is a valid entry:

```
1
00:00:10,500 --> 00:00:13,000  X1:63 X2:223 Y1:43 Y2:58
<i>Elephant's Dream</i>
```